src/html/slash.html

```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="../css/slash.css">
+    </head>
+    <body>
+        <div class="logo">
+            <p><i>Not keyed in</i></p>
```

Comment:
These strings should be pulled out into the component configuration so we can easily support i18n.

Reply:
This was just a sample page as we do not yet have designs for it. I propose that we leave it like this for now and handle internationalization later (because there a lot of labels in addition to this one that should be configurable).

Reply:
That seems reasonable to me